src/main/java/banking/CardGenerator.java

```diff
@@ -0,0 +1,57 @@
+package banking;
+
+import banking.model.Card;
+
+public class CardGenerator {
```

Comment:
Your code still is not formatted well, please follow Intellij Idea documentation.
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html

Reply:
fixed